src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/CosmosResponseProcessor.cs

```diff
@@ -163,9 +163,6 @@ private string GetCustomerManagedKeyErrorMessage(int subStatusCode)
                 case KnownCosmosDbCmkSubStatusValue.InvalidKeyVaultKeyUri:
                     errorMessage = Resources.InvalidKeyVaultKeyUri;
                     break;
-                case KnownCosmosDbCmkSubStatusValue.InvalidInputBytes:
```

Comment:
FYI: I removed this value.  Looking at new docs from https://docs.microsoft.com/en-us/rest/api/cosmos-db/http-status-codes-for-cosmosdb#http-substatus-codes, it does not seem to be related to CMK.